overlay.d/05rhcos/usr/lib/dracut/modules.d/89ibm2810xiv-mutipath-config/ibm2810xiv-multipath-config.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# Workaround for IBM 2810XIV discs
+
+enable=
+devices=$(lsblk -o name --nodeps --noheadings)
+
+for dev in ${device}; do
+    model=$(cat /sys/class/block/${dev}/device/model)
+    if [ "${model}" == "2810XIV" ]; then
+	enable=1
+	break
+    fi
+done
+
+if [ -n ${enable} ] && [ ! -e /etc/multipath.conf ] ; then
+    mkdir -p /etc/multipath/multipath.conf.d
+    mpathconf --enable
```

Comment:
This is doing what `multipathd-configure.service` does already, so couldn't we just order ourselves after that service only to run the `sed` command?